tests/MongoDB.EntityFrameworkCore.FunctionalTests/MongoDB.EntityFrameworkCore.FunctionalTests.csproj

```diff
@@ -1,30 +1,30 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-      <TargetFramework>net8.0</TargetFramework>
-        <ImplicitUsings>enable</ImplicitUsings>
-        <Nullable>enable</Nullable>
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+    <NoWarn>EF1001,CS8602,CS8618</NoWarn>
+  </PropertyGroup>
```

Comment:
Does it makes sense to migrate those settings (or a subset) to` tests/Directory.Build.props` ?

Reply:
I don't think it's worth it yet. There won't likely be more test projects or any new stuff any time soon.

Reply:
So we have `Directory.Build.props` already, seems like a natural place for such settings.